OpenOversight/app/templates/about.html

```diff
@@ -40,10 +40,9 @@
         <div class="col-lg-4">
           <div class="col-half-border">
             <span class="fa fa-map-marker fa-2x"></span>
+            <p>It is the first project of its kind in the United States, and was first implemented in Chicago in October 2016.</p>
             <p>
-              It is the first project of its kind in the United States, and was first implemented in Chicago in October 2016.
-              OpenOversight launched in the East Bay of the San Francisco Bay Area in fall 2017 and in New York City in 2018. A
-              Baltimore instance was launched in 2019 at <a href="https://bpdwatch.com">BPDWatch.com</a>.
+              There are several instances up in other areas throughout the United States: <a href="https://bpdwatch.com/" target="_blank">Baltimore</a>, <a href="https://openoversight.tech-bloc-sea.dev/" target="_blank">Seattle</a>, and <a href="https://openoversightva.org/" target="_blank">Virginia</a>.
```

Comment:
Can we change this to:
```suggestion
              There are several instances up in other areas throughout the United States: <a href="https://bpdwatch.com/" target="_blank">Baltimore</a>, <a href="https://spd.watch/" target="_blank">Seattle</a>, and <a href="https://openoversightva.org/" target="_blank">Virginia</a>.
```

Reply:
Done done done!